internal/database/sqlcommon/event_sql.go

```diff
@@ -223,5 +220,29 @@ func (s *SQLCommon) GetEvents(ctx context.Context, namespace string, filter ffap
 	}
 
 	return events, s.QueryRes(ctx, eventsTable, tx, fop, nil, fi), err
+}
+
+func (s *SQLCommon) GetEvents(ctx context.Context, namespace string, filter ffapi.Filter) (message []*core.Event, res *ffapi.FilterResult, err error) {
+
+	cols := append([]string{}, eventColumns...)
+	cols = append(cols, s.SequenceColumn())
+
+	query := sq.Select(cols...).From(eventsTable)
+
+	return s.getEventsGeneric(ctx, namespace, query, filter)
+
+}
+
+func (s *SQLCommon) GetEventsInSequenceRange(ctx context.Context, namespace string, filter ffapi.Filter, startSequence int, endSequence int) (message []*core.Event, res *ffapi.FilterResult, err error) {
+	filter.Limit(0) // We calculate the limit below using the sequence IDs
+
+	cols := append([]string{}, eventColumns...)
+	cols = append(cols, s.SequenceColumn())
+
+	query := sq.Select(cols...).FromSelect(sq.Select(cols...).From(eventsTable).Where(sq.GtOrEq{
```

Comment:
Calling this out as something potentially worth of discussion, initially I wanted to use the `BETWEEN` operator referencing the sequence ID but squirrel (the underlying SQL construction lib we're using) does not support building queries with `BETWEEN`.

The other option was to use a sub-query and then apply a limit and offset but this was _not good_ from a performance perspective, so I've gone for a sub-query and a `WHERE` which _should_ jump straight to the correct sequence ID. Comparing the use of OFFSET and this approach showed roughly a ~140ms difference in speed.

~440ms for a query using OFFSET, searching for 1000 records between sequence ID 100,000 and 101,000
~300ms for a query using WHERE, searching for 1000 records between sequence ID 100,000 and 101,000

Reply:
In discussion w/ @nguyer there was no longer any point in using a sub query here (not that it made much sense in the first place) so this is swapped out for a way more performant query in [b6cdd32](https://github.com/hyperledger/firefly/pull/1452/commits/b6cdd3295ac1bcd0a2856712959aeb7c2b1c4af4).

Looking now at ~50ms for a response on my machine using postman and SQLite.